concepts/symbols/.meta/config.json

```
@@ -0,0 +1,5 @@
+{
+  "blurb": "Symbols are identifiers and are created through the symbol literal. They are non mutable, which makes them useful for things like keys in a hashes.",
```

Symbols are these things, but they are used as identity.  They are, of course, used as keys for Hash objects, but we could mention that they are used as the internal names for constants (including module and class names), variables, method names, and keywords.  They are not only immutable, but they are singletons, as well.

---

Feel free to suggest another blurb/modified one.

---

```suggestion
  "blurb": "Symbols are very important in Ruby.  They are used as "identity".  They are immutable, guaranteed to be a single item no matter where it is used or referenced.  They are used internally as variable, method, constants, and keyword identifiers.  They are also useful, and often used, as keys in a `Hash`."
```

Something like that.  Hopefully that helps set up the expectation that these things will be covered, or demonstrated, during the concept.